backend/src/main/resources/data.sql

```diff
@@ -46,4 +46,16 @@ VALUES (1, 'https://i.ebayimg.com/images/g/nMEAAOSwZ6lixPMT/s-l500.png', 1),
        (6, 'https://i.ebayimg.com/images/g/hfkAAOSwXBdis81I/s-l500.jpg', 6),
        (7, 'https://i.ebayimg.com/images/g/Yl8AAOSwB3BaCF~W/s-l500.jpg', 7),
        (8, 'https://i.ebayimg.com/images/g/X2MAAOSwY1Vhk9Yo/s-l500.jpg', 8),
-       (9, 'https://i.ebayimg.com/images/g/HfcAAOSwOAViGkIb/s-l500.jpg', 9)
```

Comment:
Can we add products that have more than one picture in data.sql ? So we can check picture switch feature.